### DISCUSSION TITLE
能否增加检测访问的IP/域名，自动选择局域网模式或者公网模式

### DISCUSSION THREAD

**Discussion by xwndo** [hslr-s/sun-panel#169] [Ideas]
就是比如，使用内网192.168.1.1这种地址访问，就会默认自动选择内网地址跳转，如果使用公网域名访问，就自动使用公网地址跳转。同时这个选项可以设置开关的这样。

**Comment by czxjh17** [comment]
支持哈，顺便可以让只有内网链接的项目隐藏，这个其实算是公网和内网切换的升级版吧。要是有朋友访问，但又不需要内网的链接，这个功能就能很好的减少显示出来的项目，让一些只有自己用的项目不公开显示。希望考虑这个建议。

**Comment by Yaidea** [comment]
这个必须支持，要是能换成后台管理地址就更好了，不用在右上角显示头像了
